File: JavaScript/8js 3/switchItUp.js
/*
|--------------------------------------------------------------------------
| Switch it Up!
|--------------------------------------------------------------------------
|
*/

function switchItUp(number) {
   if (number == 1) {
      return 'One'
   } else if (number == 2) {
      return 'Two'
   } else if (number == 3) {
      return 'Three'
   } else if (number == 4) {
      return 'Four'
   } else if (number == 5) {
      return 'Five'
   } else if (number == 6) {
      return 'Six'
   } else if (number == 7) {
      return 'Seven'
   } else if (number == 8) {
      return 'Eight'
   } else if (number == 9) {
      return 'Nine'
   } else {
      return 'Zero'
   }
}


function switchItUp(n) {
   return ["Zero", "One", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine"][n]
}


function switchItUp(number) {
   return 'Zero One Two Three Four Five Six Seven Eight Nine'.split(' ')[number];
}


function switchItUp(number) {
   switch (number) {
      case 0:
         return 'Zero';
      case 1:
         return 'One';
      case 2:
         return 'Two';
      case 3:
         return 'Three';
      case 4:
         return 'Four';
      case 5:
         return 'Five';
      case 6:
         return 'Six';
      case 7:
         return 'Seven';
      case 8:
         return 'Eight';
      case 9:
         return 'Nine';
   }
}


console.log(switchItUp(1), "One");
console.log(switchItUp(3), "Three");
console.log(switchItUp(5), "Five");

File: Python/8py 2/switch_it_up.py
"""
|--------------------------------------------------------------------------
| Switch it Up!
|--------------------------------------------------------------------------
|
"""

def switch_it_up(number):
   if number == 1:
      return 'One'
   elif number == 2:
      return 'Two'
   elif number == 3:
      return 'Three'
   elif number == 4:
      return 'Four'
   elif number == 5:
      return 'Five'
   elif number == 6:
      return 'Six'
   elif number == 7:
      return 'Seven'
   elif number == 8:
      return 'Eight'
   elif number == 9:
      return 'Nine'
   else:
      return 'Zero'


def switch_it_up(n):
   return ['Zero','One','Two','Three','Four','Five','Six','Seven','Eight','Nine'][n]


def switch_it_up(number):
   number_converter={0:"Zero",1:"One",2:"Two",3:"Three",4:"Four",5:"Five",6:"Six",7:"Seven",8:"Eight",9:"Nine"}
   return number_converter[number]


def switch_it_up(number):
   dict = {
   1: "One",
   2: "Two",
   3: "Three",
   4: "Four",
   5: "Five",
   6: "Six",
   7: "Seven",
   8: "Eight",
   9: "Nine",
   0: "Zero"}
   return dict.get(number)


print(switch_it_up(0), "Zero")
print(switch_it_up(9), "Nine")